### ISSUE TITLE
Folder Won't Delete

### ISSUE THREAD

**Issue by Togora** [pbek/QOwnNotes#374]
I have downloaded and unzipped QOwnNotes (ver 16.10.8) in my Downloads folder. On trying to move it to another folder I find it won't move, also if I try to delete the folder it won't delete.

I have checked in Task Manager to see what processes are running and I can't see anything untoward. Also I have carried out a restart of my laptop but to no avail.

Point to note, I don't have this issue with any other software, only QOneNote.

Hope this helps

**Comment by pbek** [comment]
Thank you for your report, but I'm not sure if this is an issue of QOwnNotes since you are not running it. Maybe the file permission are "wrong"? Can you test moving the folder in a terminal?

**Comment by Togora** [comment]
Thanks for the quick response.

I have managed to resolve the problem by going into "Safe Mode" and deleting QOneNotes that way. I did notice in Task Manager that there was a download action taking place as I was trying to delete the folder, now that is odd.

By the way I did try deleting the folder via an Admin Terminal but it made no difference and just hung, this was after checking all permissions were as expected.

Lastly, in case you are wondering, I always download QOneNote using the links you provide in your Tweets and I scan the files twice, once zipped and then unzipped, just in case a packed file hides something nasty.

Hope this helps
